### PR TITLE
[16.0] [ADD] partner_subject_to_vat

### PR DESCRIPTION
**Pull request by victor-champonnois**
New module adding a boolean fields to record whether the partner is subject to VAT. It does not affect the visibility of the VAT because the VAT information could be useful anyway.

### CODE REVIEW

File: partner_subject_to_vat/models/res_partner.py
Comment (carmenbianca) [inline partner_subject_to_vat/models/res_partner.py]:
```suggestion
    is_subject_to_vat = fields.Boolean(string="Is Subject to VAT")
```

**Review by robinkeunen — APPROVED** [review]

File: partner_subject_to_vat/__manifest__.py
Comment (carmenbianca) [inline partner_subject_to_vat/__manifest__.py]: not needed

File: partner_subject_to_vat/__manifest__.py
Comment (carmenbianca) [inline partner_subject_to_vat/__manifest__.py]: keys with empty values can be deleted

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Small suggestions, otherwise LGTM.

I leave it to the PSC to determine whether this module belongs in `partner-contact`

File: partner_subject_to_vat/__manifest__.py
Comment (victor-champonnois) [reply partner_subject_to_vat/__manifest__.py]: fixed

File: partner_subject_to_vat/__manifest__.py
Comment (victor-champonnois) [reply partner_subject_to_vat/__manifest__.py]: fixed

File: partner_subject_to_vat/models/res_partner.py
Comment (victor-champonnois) [reply partner_subject_to_vat/models/res_partner.py]: fixed

**Review by carmenbianca — APPROVED** [review]

**Comment by victor-champonnois** [comment]
@gurneyalex @simahawk @bguillot Can this be merged ?

**Comment by carmenbianca** [comment]
@OCA/crm-sales-marketing-maintainers Kindly pinging for a merge/review :)

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

**Pull request merged**
